blog/2019-12-20-infinispan-10.1.0.Final.adoc

```diff
@@ -0,0 +1,87 @@
+---
+layout: blog
+title: Infinispan 10.1.0.Final
+date: '2019-12-20T12:00:00.000-00:00'
+author: Tristan Tarrant
+tags: [ "release" ]
+---
+
+= Infinispan 10.1 "Turia"
+
+Hi there,
+
+we finish 2019 in style with the Final release of Infinispan 10.1, codenamed https://www.cervezaturia.es["Turia"].
+
+== Server console
+
+The highlight of this release is the new server console which is now based on https://www.patternfly.org/v4/[Patterfly 4] and https://www.patternfly.org/v4/[React.js].
```

Comment:
the link for Reac.js is pointing to patternfly webapage.
shouldn't be https://reactjs.org/ ?

Reply:
Fixed